### ISSUE TITLE
immediately after opening .pde file, all tabs marked as modified

### ISSUE THREAD

**Issue by benfry** [processing/processing#4786]
Believe this was using File > Open to load something not part of the sketchbook. All tabs immediately lit up as modified. Feels like an editor regression with the undo or file modified changes.

<img width="564" alt="161212 open sketch everything modified" src="https://cloud.githubusercontent.com/assets/1623101/21406818/7dda4e86-c79b-11e6-9eb3-29a8374e8e5c.png">

**Comment by JakubValtar** [comment]
@benfry Do you have any breakpoints in the code?

I found a bug in undo, after undo/redo all tabs with breakpoints will be marked as modified, because SketchCode.savedProgram has extra comments for breakpoints and comparing it with tab contents will always show as not equal...

I can't think about any other reason for this.

**Comment by benfry** [comment]
Nope. Unfortunately it was a random thing, I took a screenshot, and I'll see if it happens again. Just wanted to flag it in case we see it again.

**Comment by JakubValtar** [comment]
Was it 3.2.3 or github master?

**Comment by benfry** [comment]
master
